librarians/urls.py

```diff
@@ -16,6 +16,9 @@
 from django.contrib import admin
 from django.urls import path
 
+from librarians.core.views import redirect_to_admin
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', redirect_to_admin),
```

Comment:
This redirect is temporary? Otherwise we can just use empty path for admin.site.urls

Reply:
is temporary, I'll change this